src/tests/__snapshots__/AddChartIcon.html

```diff
@@ -1 +1 @@
-<span data-v-2fd133ee="" class="kui-icon add-chart-icon" data-testid="kui-icon-wrapper-add-chart-icon" style="box-sizing: border-box; color: rgb(0, 68, 244); display: inline-flex; flex-shrink: 0; height: 32px; line-height: 0; width: 32px;"><svg data-v-2fd133ee="" data-testid="kui-icon-svg-add-chart-icon" fill="none" height="100%" role="img" viewBox="0 0 24 24" width="100%" xmlns="http://www.w3.org/2000/svg"><title data-v-2fd133ee="" data-testid="kui-icon-svg-title">My custom title</title><path data-v-2fd133ee="" d="M5 21C4.45 21 3.97917 20.8042 3.5875 20.4125C3.19583 20.0208 3 19.55 3 19V5C3 4.45 3.19583 3.97917 3.5875 3.5875C3.97917 3.19583 4.45 3 5 3H14V5H5V19H19V10H21V19C21 19.55 20.8042 20.0208 20.4125 20.4125C20.0208 20.8042 19.55 21 19 21H5ZM7 17H9V10H7V17ZM11 17H13V7H11V17ZM15 17H17V13H15V17ZM17 9V7H15V5H17V3H19V5H21V7H19V9H17Z" fill="currentColor"></path></svg></span>
\ No newline at end of file
+<span data-v-2fd133ee="" class="kui-icon add-chart-icon" data-testid="kui-icon-wrapper-add-chart-icon" style="box-sizing: border-box; color: rgb(0, 68, 244); display: inline-flex; flex-shrink: 0; height: 32px; line-height: 0; width: 32px;"><svg data-v-2fd133ee="" data-testid="kui-icon-svg-add-chart-icon" fill="none" height="100%" role="img" viewBox="0 0 24 24" width="100%" xmlns="http://www.w3.org/2000/svg"><title data-v-2fd133ee="" data-testid="kui-icon-svg-title">My custom title</title><!-- eslint-disable-next-line vue/no-v-html --><g data-v-2fd133ee=""><path d="M5 21C4.45 21 3.97917 20.8042 3.5875 20.4125C3.19583 20.0208 3 19.55 3 19V5C3 4.45 3.19583 3.97917 3.5875 3.5875C3.97917 3.19583 4.45 3 5 3H14V5H5V19H19V10H21V19C21 19.55 20.8042 20.0208 20.4125 20.4125C20.0208 20.8042 19.55 21 19 21H5ZM7 17H9V10H7V17ZM11 17H13V7H11V17ZM15 17H17V13H15V17ZM17 9V7H15V5H17V3H19V5H21V7H19V9H17Z" fill="currentColor"></path></g></svg></span>
```

Comment:
Added an eslint rule rather than inlining the comment ignoring `v-html`. Since this repo is so specialized and only contains a single component, there's no risk here